server/command_import.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"encoding/json"
+	"strings"
+
+	cloud "github.com/mattermost/mattermost-cloud/model"
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/pkg/errors"
+)
+
+func (p *Plugin) runImportCommand(args []string, extra *model.CommandArgs) (*model.CommandResponse, bool, error) {
+	if len(args) == 0 {
+		return nil, true, errors.New("must provide an installation DNS")
+	}
+	installationDNS := standardizeName(args[0])
+
+	splitDNS := strings.Split(installationDNS, ".")
+	if len(splitDNS) < 2 {
+		return nil, true, errors.New("failed to parse DNS value")
+	}
+	name := splitDNS[0]
+
+	cloudInstall, err := p.cloudClient.GetInstallationByDNS(installationDNS, nil)
+	if err != nil {
+		return nil, false, errors.Wrap(err, "failed to get installation by DNS")
+	}
+	if cloudInstall == nil {
+		return nil, true, errors.New("no installation for the DNS provided")
+	}
```

Comment:
We need to check one other thing after this. We need to make sure that the installation hasn't already been imported by someone else in the cloud plugin.

To do this, we should see if the installation ID is found in the plugin database.